lib/twemoji.rb

```diff
@@ -196,8 +235,12 @@ def self.has_ancestor?(node, tags)
     # @return [String] Returns a String just like content with all emoji text
     #                  replaced by the corresponding emoji image.
     # @private
-    def self.filter_emojis(content)
-      content.gsub(emoji_pattern) { |match| img_tag(match) }
+    def self.filter_emojis(content, pattern=emoji_pattern)
```

Comment:
If we want these class methods to be private, need to use `private_class_method :filter_emojis`<sup>[Source](http://ruby-doc.org/core-2.3.3/Module.html#method-i-private_class_method)</sup>